### ISSUE TITLE
Add filtering by ruleset to List of all violations

### ISSUE THREAD

**Issue by nekennedy** [daisy/ace#109]
The **Summary of violations, organized by ruleset and severity** is great. 

Some people may want to begin with WCAG A issues when they start tackling accessibility, and the ability to filter by those in the **List of all violations** could be helpful for those who don't know which rules belongs to which ruleset.
